### PR TITLE
google-app-login

### PR DESCRIPTION
**Pull request by RashmiDholakiya**
Implement Login with google functionality.

### CODE REVIEW

File: app/Http/Controllers/Auth/AuthController.php [hunk 1]
```diff
@@ -3,65 +3,107 @@
...
+		}
+
+		$authUser = $this->findOrCreateUser($user);
+
+		Auth::login($authUser, true);
+
+		return Redirect::to('/');
+	}
+
+	private function findOrCreateUser($googleUser)
```
Comment (poojajajal) [inline app/Http/Controllers/Auth/AuthController.php]: For variable use snack case.
And for function use camel case.

File: app/Http/Controllers/Auth/AuthController.php [hunk 2]
```diff
@@ -3,65 +3,107 @@
...
+	{
+
+		$authUser = User::where('google_id', $googleUser->id)->first();
+
+		if ($authUser) {
+			return $authUser;
+		}
+
+		return User::create([
```
Comment (poojajajal) [inline app/Http/Controllers/Auth/AuthController.php]: Here you can create user object and fill the value and save it.

File: app/Http/Controllers/Auth/AuthController.php (hunk 1 above)
Comment (RashmiDholakiya) [inline app/Http/Controllers/Auth/AuthController.php]: Done

File: app/Http/Controllers/Auth/AuthController.php (hunk 2 above)
Comment (RashmiDholakiya) [inline app/Http/Controllers/Auth/AuthController.php]: Done

**Comment by poojajajal** [comment]
Code is look good now. 
@RashmiDholakiya  first resolve conflict then after you can merge it .

**Comment by RashmiDholakiya** [comment]
Implement Google Apps login

**Comment by RashmiDholakiya** [comment]
Implement Google-App Login

**Pull request merged**